apps/right-to-rent-check/translations/src/en/fields.json

```diff
@@ -24,6 +24,37 @@
   "tenant-country": {
     "label": "Country of nationality"
   },
+  "tenant-additional-details": {
+    "legend": "Additional details",
+    "options": {
+      "reference-number": {
+        "label": "<h3>Home Office reference number</h3>If the person has applied for a visa with the Home Office, they will be issued with a reference number. This could be an application or appeal number, an Application Registration Card (ARC) number or a case ID number."
```

Comment:
I'm not thrilled about this - putting an `<h3>` in the label, but it has the desired effect.